src/app/pages/Sandbox/Editor/Workspace/Project/index.js

```diff
@@ -155,26 +147,23 @@ export default class Project extends React.PureComponent {
               </Link>
             </Item>
           </div>}
-        <WorkspaceSubtitle>Statistics</WorkspaceSubtitle>
-        <ViewCountContainer>
-          <ViewCount>{likeCount}{' '}</ViewCount>
-          <ViewCountDescription>
-            {likeCount === 1 ? ' like' : ' likes'}
-          </ViewCountDescription>
-        </ViewCountContainer>
-        <ViewCountContainer>
-          <ViewCount>{forkCount}{' '}</ViewCount>
-          <ViewCountDescription>
-            {forkCount === 1 ? ' fork' : ' forks'}
-          </ViewCountDescription>
-        </ViewCountContainer>
-        <ViewCountContainer>
-          <ViewCount>{viewCount}{' '}</ViewCount>
-          <ViewCountDescription>
-            unique
-            {viewCount === 1 ? ' view' : ' views'}
-          </ViewCountDescription>
-        </ViewCountContainer>
+
+        {githubUrl &&
+          <div>
+            <WorkspaceSubtitle>Source Control</WorkspaceSubtitle>
+            <Item>
+              <a href={githubUrl} rel="noopener noreferrer" target="_blank">
+                {repoName}
+              </a>
+            </Item>
+          </div>}
+
+        <Statistics
+          viewCount={viewCount}
```

Comment:
Hmm you're right, do this would also clash with the boolean implicit =true. Interesting